lnbits/core/templates/admin/index.html

```diff
@@ -406,6 +441,83 @@
           m => m !== manifest
         )
       },
+      toggleServerLog() {
+        this.serverlogEnabled = !this.serverlogEnabled
+        if (this.serverlogEnabled) {
+          const wsProto = location.protocol !== 'http:' ? 'wss://' : 'ws://'
+
+          async function digestMessage(message) {
```

Comment:
- maybe this generic function should go in `lnbits/static/js/base.js`
- or in a fresh `lnbits/static/js/utils.js`
- it might be used in different places in the future